src/templates/static/special/rebootLocalHost.yml

```diff
@@ -0,0 +1,21 @@
+# Reboots a system if any of the preceeding plays sets the 'insights_needs_reboot' variable to true.
+# The variable can be overridden to suppress this behavior.
+- name: Reboot localhost
+  hosts: "@@HOSTS@@"
+  vars:
+    insights_signature_exclude: "/hosts"
+  become: True
+  gather_facts: False
+  tasks:
+    - when:
+        - insights_needs_reboot is defined
+        - insights_needs_reboot
+      block:
+        - name: Reboot system
+          shell: sleep 2 && shutdown -r +1 now "Ansible triggered reboot"
+          async: 1
+          poll: 0
+          ignore_errors: true
+        - name: Schedule insights-client after boot
+          shell: touch /.run_insights_client_next_boot
```

Comment:
Using literaly `/.run_insights_client_next_boot` seems odd. Is there a more appropriate place to put this? E.g. insights-client configuration directory?

Reply:
This is a good question.  I'll reach out to Link about it to see what I can come up with.

Reply:
@jharting After some discussion with Link the more appropriate place for this file would have to be **/etc/insights-client**.  I'll make that change in this next iteration.